docs/supported_platforms.md

```diff
@@ -30,6 +30,24 @@ The following Intel feature technologies are supported in the 1.0.1 release.
 | Intel® Data Center GPU Flex Series            | [Intel Data Center GPU Flex 140](https://www.intel.com/content/www/us/en/products/sku/230020/intel-data-center-gpu-flex-140/specifications.html) <br/>  [Intel Data Center GPU Flex 170](https://www.intel.com/content/www/us/en/products/sku/230019/intel-data-center-gpu-flex-170/specifications.html)  |
 | Intel® QuickAssist Technology (Intel® QAT) | [4th Gen Intel® Xeon® Scalable processors](https://www.intel.com/content/www/us/en/products/docs/processors/xeon-accelerated/4th-gen-xeon-scalable-processors.html) |                               
 
+## Setting up Intel Hardware Features
+
+### BIOS Configuration
+Note: Please refer to your BIOS vendor for specific instructions. This is only a reference for BIOS configuration. 
+| Feature | BIOS Configuration | 
+| ----- | ---------------------- |
+| SGX | [Link](https://www.intel.com/content/www/us/en/support/articles/000087972/server-products/single-node-servers.html) |
```

Comment:
Can you keep the 'Feature' column consistent in both tables.. You can keep it similar to Supported RHOCP versions table's feature column

Reply:
Good catch! Thanks for the review. Done, updated PR!